tests/vision-camera/vision-camera.js

```diff
@@ -11,18 +35,55 @@ export class mockCamera extends React.PureComponent {
     ];
   }
 
+  componentDidUpdate() {
+    const { frameProcessor } = this.props;
+    frameProcessor?.frameProcessor( mockFrame );
+  }
+
   // eslint-disable-next-line class-methods-use-this, react/no-unused-class-component-methods
   async takePhoto( ) {
-    const writePath = `${RNFS.DocumentDirectoryPath}/simulated_camera_photo.png`;
-
-    const imageDataBase64 = "some_large_base_64_encoded_simulated_camera_photo";
-    await writeFile( writePath, imageDataBase64, "base64" );
-
-    return { path: writePath };
+    // TODO: this only works on iOS
+    return CameraRoll.getPhotos( {
+      first: 20,
+      assetType: "Photos"
+    } )
+      .then( async r => {
+        console.log( "r.edges", r.edges );
+        const testPhoto = r.edges[r.edges.length - 1].node.image;
```

Comment:
Basically, we are reading the newest twenty photos from the simulators gallery and return the oldest one of those.